e2e/paths/enterRespondents_test.js

```diff
@@ -8,13 +8,20 @@ Before((I, caseViewPage) => {
   caseViewPage.goToNewActions(config.applicationActions.enterRespondents);
 });
 
-Scenario('Entering information for respondent and submitting', (I, enterRespondentsPage) => {
+Scenario('Entering information for respondent and submitting', (I, enterRespondentsPage, caseViewPage) => {
   enterRespondentsPage.enterRespondent('firstRespondent', respondent);
   I.continueAndSubmit(config.eventSummary, config.eventDescription);
   I.seeEventSubmissionConfirmation(config.applicationActions.enterRespondents);
+  caseViewPage.selectTab(caseViewPage.tabs.casePeople);
+  I.seeAnswerInTab(1, 'Respondent 1', 'Full name', 'Joe Bloggs');
```

Comment:
To be honest in the future I would remove checks for tab content from `half complete` tests to avoid duplications. Makes sense to test it only once when all data is populated.